### ISSUE TITLE
LiveQuery not working for Safari

### ISSUE THREAD

**Issue by cheesykyle** [ParsePlatform/Parse-SDK-JS#261]
LiveQueries don't seem to work with the current version of Safari(9.1). I am unable to subscribe to queries and nothing appears in the parse-server log, however it works fine in Chrome and the current Safari Technology Preview. Is this a known issue and are there any workarounds for this?

**Comment by cheesykyle** [comment]
Is anyone using Live Query able to confirm if Safari is having issues? Currently trying to determine if it is a bug in the current version of Safari itself.

**Comment by alexcontanu** [comment]
I found the problem in Safari:
Line 1311: `return typeof WebSocket === 'function' return false`
In Safari typeof WebSocket is `object`.
I tested `return typeof WebSocket === 'function' || typeof WebSocket === 'object'`and works

**Comment by cheesykyle** [comment]
@alexcontanu Which version of the SDK are you using? I should've specified, I'm on the current 1.8.3. There is no line in 1.8.3 containing: `return typeof WebSocket === 'function' return false`

**Comment by alexcontanu** [comment]
Parse JavaScript SDK v1.8.3

```
{
    key: '_getWebSocketImplementation',
    value: function _getWebSocketImplementation() {
      return typeof WebSocket === 'function' || typeof WebSocket === 'object' ? WebSocket : null;
    }

    // ensure we start with valid state if connect is called again after close
  }
```
From npm:
https://www.npmjs.com/package/parse

**Comment by cheesykyle** [comment]
That did the trick!

**Comment by cheesykyle** [comment]
I've made a pull request for this issue. Thanks for the fix! #266
